fusionfrontend/src/app/fleet/components/content/asset-series-create/view-mode.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum ViewMode {
```

Comment:
Just an refactoring idea: Maybe we could use the same enum for all Dialogs and Wizards (at least per module)